main.go

```diff
@@ -52,23 +51,18 @@ func runScript(runner string, filePath string, workingDir string) (int, error) {
 		binary = splitRunner[0]
 		paramList = splitRunner[1:]
 	} else {
-		return 1, fmt.Errorf("Error: %s", err.Error())
+		return 1, fmt.Errorf("shell quote split errpr: %s", err.Error())
 	}
 
 	paramList = append(paramList, filePath)
 
-	script := command.NewWithStandardOuts(binary, paramList...).SetStdin(os.Stdin).SetDir(workingDir)
+	script := command.New(binary, paramList...).SetDir(workingDir)
 
-	exitCode, err := script.RunAndReturnExitCode()
+	out, err := script.RunAndReturnTrimmedCombinedOutput()
 	if err != nil {
-		errorString := fmt.Sprintf("Error: %s, command: %#v", err.Error(), script.GetCmd())
-
-		if exitCode == 0 {
-			return 1, errors.New(errorString)
-		}
-
-		return exitCode, errors.New(errorString)
+		exitCode := script.GetCmd().ProcessState.ExitCode()
```

Comment:
I think we should differentiate if the error is an exit error or another error.
exit status only makes sense in the case of exit errors.
this is how this can be tested:
```
	var exitErr *exec.ExitError
	if errors.As(err, &exitErr) {
            // exit error
        }
```
I also suggest constructing error messages for command executions based on this: https://bitrise.atlassian.net/wiki/spaces/RD/pages/2445082990/Human+readable+error+messages+for+Steps#4,-Fixed-structure-for-command-execution-errors